XenModel/XenSearch/Common.cs

```diff
@@ -646,7 +650,7 @@ private static IComparable MemoryTextProperty(IXenObject o)
                         vm.power_state != vm_power_state.Running)
                         return null;
 
-                    if (vm.virtualisation_status != VM.VirtualisationStatus.OPTIMIZED)
+                    if (!vm.GetVirtualisationStatus.HasFlag(VM.VirtualisationStatus.IO_DRIVERS_INSTALLED | VM.VirtualisationStatus.MANAGEMENT_INSTALLED))

```

Comment:
I was hoping that all the instances like this would need only one of IO_DRIVERS_INSTALLED and MANAGEMENT_INSTALLED. Why would it need both?